### PR TITLE
Add a tip regarding the Snakemake workflow visualization

### PR DESCRIPTION
**Pull request by ekatef**
Integrates tips regarding #17 into the tutorial

### CODE REVIEW

File: 3_how_to_execute_the_workflow.ipynb [hunk 1]
```diff
@@ -84,7 +84,9 @@
...
+    "You should see now a image \"Directed Acyclic Graph\" or short \"dag\" in your folder which is saved as **workflow.png** and describes the workflow that you would execute. Just image you would need to execute script by script by hand. Not impossible for the workflow below, but just imagine you have 100 or 1000 optimisation. Snakemake saves your valuable time in such situations.\n",
+    "\n",
+    "Sometimes visualization with the directive above may result in an error like `'Error: <stdin>: syntax error in line 1 near 'Restricted''`. (Big thanks to Yiliao Zhou for reporting this issue!) This error appears if the Snakemake output contains some text like solver info messages which is not supposed to be there by the graphiz visualizer. The solution is to get rid of this additional text. For example, we may cut a part of the output which is meaningful for building a workflow graph with `snakemake --dag solve_all_networks | sed -n \"/digraph/,/}/p\" | dot -Tpng -o workflow.png`."
```
Comment (pz-max) [inline 3_how_to_execute_the_workflow.ipynb:89]: Looks good! Maybe reference the issue. We discussed there a bit. Thanks for adding this. The community will appreciate that :)

Comment (ekatef) [reply 3_how_to_execute_the_workflow.ipynb]: Done :)

According to my impression, it was a great idea to organise the hackathon and it absolutely makes sense to keep it's materials updated. Than you so much for driving this!

**Comment by pz-max** [comment]
@ekatef Thanks Katia & congrats to the next merged PR :)

**Comment by ekatef** [comment]
@pz-max Happy to contribute :)

**Pull request merged**